### DISCUSSION TITLE
Why does `z.string().datetime()` disallow timezone offsets by default?

### DISCUSSION THREAD

**Discussion by mtsknn** [colinhacks/zod#2793] [Q&A]
What's the rationale for `z.string().datetime()` to disallow timezone offsets by default? (Offsets can be allowed by setting the `offset` option to `true`.)

The only "rationale" I found is from the ["RFC for datetime methods" comment](https://github.com/colinhacks/zod/issues/126#issuecomment-708589382) by @colinhacks from 2020:

> I don't like AJV's datetime regex since it allows timezone offsets, which should almost never ever be used imo.

But why should timezone offsets almost never be used?

**Comment by JacobWeisenburger** [comment]
+1
